webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require("path");
+
+console.info("Building in " + process.env.NODE_ENV + " mode.");
+module.exports = {
+  mode: process.env.NODE_ENV ? "production" : "development",
+  entry: {
+    index: "./lib/index.ts",
+  },
+  devtool: process.env.NODE_ENV ? "source-map" : "inline-source-map",
+  output: {
+    library: {
+      name: "helpers",
+      type: "umd",
+    },
+  },
+  module: {
+    rules: [
+      {
+        test: /\.([cm]?ts|tsx)$/,
```

Comment:
```suggestion
        test: /\.ts$/,
```
Let's keep it simple. We don't use anything other than `ts`, so there's no need to have the loader cope with anything else.